docs/adr/adr-009-public-api.md

```diff
@@ -209,6 +209,15 @@ type FraudModule interface {
 }
 ```
 
+##### Metrics
+
+```go
+type MetricsModule interface {
+ // TODO	
+}
+```
```

Comment:
Can you elaborate on this module pls? I don't really get its purpose and don't see how its API would look like

Reply:
So now it's much clear what it should be doing :)